### DISCUSSION TITLE
Tag Groups

### DISCUSSION THREAD

**Discussion by bascherz** [joomla/joomla-cms#31605] [2. Cold Features]
Tags sometimes need context. Article categories already provide that. But tags are only defined in bulk. It would be awesome to be able to associate groups of tags with article categories, like field groups do for custom fields, so that only the tags applicable to those categories can be selected. This will reduce the size of the list of tags available to any given article.

### Steps to reproduce the issue
NOT AN ISSUE.

### Expected result
Better organization and association of tags.

### Actual result
NONE

### System information (as much as possible)
Joomla 3.9.23

### Additional comments
This just seems like an obvious thing to do (to me anyway).

**Comment by ghost** [comment]
@bascherz new features go in Joomla4, mean 4.1 if someone code a pull request, it get successfully tested by 2 people and its accepted by maintainer.
